src/server/enrollment-system.js

```diff
@@ -365,7 +366,7 @@ function veteranToSpouseFinancials(veteran) {
     grossIncome: veteran.spouseGrossIncome,
     netIncome: veteran.spouseNetIncome,
     otherIncome: veteran.spouseOtherIncome
-  });
+  }, { grossIncomeType: 7 });
```

Comment:
ignore this - leftover from when I thought we had to override the types based on dependent/spouse. Will push up a new diff shortly.